.github/workflows/test.yml

```diff
@@ -161,6 +161,7 @@ jobs:
         shell: bash
         run: |
           set -e
+          rm -rf examples/ffi/build_dll
           mkdir examples/ffi/build_dll
           cd examples/ffi/build_dll
           cmake -S .. -DDatadog_ROOT=$LIBDD_OUTPUT_FOLDER -DVCRUNTIME_LINK_TYPE=DLL
```

Comment:
To be honest I do not know. But I observed FFI steps failing: `symbolizer` could not run because `libdatadog_profiling_ffi.so` was missing (there is no `libdatadog_profiling_ffi.so` anymore, it's renamed into `libdatadog_profiling.so`

When I added this line, the issue went away.

Reply:
This looks suspiciously exactly like the issue @AlexJF ran into in https://github.com/DataDog/libdatadog/pull/408 ...